api/woeip/apps/air_quality/serializers.py

```diff
@@ -56,6 +56,19 @@ class Meta:
         model = Collection
         fields = ["starts_at", "ends_at", "collection_files"]
 
+    def create(self, validated_data):
+        files_data = self.context.get('view').request.FILES
+        collection = Collection.objects.create(
+            starts_at=validated_data.get('starts_at'),
+            ends_at=validated_data.get('ends_at'),
+        )
+        for file in files_data:
+            CollectionFile.objects.create(
+                collection=collection,
+                file=file,
+            )
+        return collection
```

Comment:
@nbilenko, what do you think about using a map function to write a bit cleaner code?

```python
def create(self, validated_data):
    collection = Collection.objects.create(**validated_data)
    files = self.context.get('view').request.FILES
    map(lambda file: self.__create_files(collection, file), files)
    return collection

def __create_files(self, collection, file):
    CollectionFile.objects.create(
        collection=collection,
        file=file
    )
```